spooky-oysters/React-FrontEnd
# Delete button

## Link to Ticket

no direct ticket

## Description of Proposed Changes

- added delete function
- added ability to pass in the id of the training program to use dynamically in the api call
- added error handling to the delete function
- reformatted the trainingProgram details

## Steps to Test

Outline the steps to test

```sh
git fetch --all
git checkout deleteButton
npm start
```

In Browser, navigate to:
http://bangazon.com:3000/

Steps to test:
Click 'Add New' and add a new test program that has a start date in the future
Press the delete button and see if you get a success message in the DOM and a 200 message in the network tab of Chrome Dev Tools next to the DELETE api call.



## Impacted Areas in Application

List general components of the application that this PR will affect:

* Details.js

## Mentions @username

Tag users that need to review this code
